app/gateway/2.7.x/vitals/vitals-estimates.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Estimate Vitals Storage in PostgreSQL
+badge: enterprise
+---
+
+Vitals data can be divided into two categories: Kong Gateway node statistics and request response codes.
+
+## Kong Gateway node statistics
+
+These types of metrics are proxy latency, upstream latency, and cache hit/miss. Kong Gateway node statistics are stored in tables like the following:
+
+* `vitals_stats_seconds_timestamp` stores 1 new row for every _second_ Kong runs
+* `vitals_stats_minutes` stores 1 new row for every _minute_ Kong runs.
+* `vitals_stats_days` stores 1 new row for every _day_ Kong runs.
+
+Kong Gateway node statistics are not associated with specific Kong Gateway entities like Workspaces, Services, or Routes. They're designed to represent the cluster's state in time. This means the tables will have new rows regardless if Kong Gateway is routing traffic or idle.
+
+The tables do not grow infinitely and hold data for the following duration of time:
+
+* `vitals_stats_seconds_timestamp` holds data for 1 hour (3600 rows)
+* `vitals_stats_minutes` holds data for 25 hours (90000 rows)
+* `vitals_stats_days` holds data for 2 years (730 rows)
+
+## Request response codes
+
+Request response codes are stored in the other group of tables following a different rationale. Tables in this group share the same structure (`entity_id`, `at`, `duration`, `status_code`, `count`):
+
+* `vitals_code_classes_by_workspace`
+* `vitals_code_classes_by_cluster`
+* `vitals_codes_by_route`
+
+The `entity_id` does not exist in `vitals_code_classes_by_cluster` as it doesn't store per any entity.
```

Comment:
"doesn't store per any entity" is confusing to me. I'm guessing that it's meant to say something like "this table doesn't store entity-specific information".